Dangerfile

```diff
@@ -11,20 +12,20 @@ def fail_if_no_supported_label_found
     markdown <<-MARKDOWN
   | Label | Description |
   |-------|-------------|
-  | *breaking* | Changes that are breaking |
-  | *build* | Changes that affect the build system |
-  | *ci* | Changes to our CI configuration files and scripts |
-  | *docs* | Documentation only changes |
-  | *feat* | A new feature |
-  | *fix* | A bug fix |
-  | *perf* | A code change that improves performance |
-  | *RevenueCatUI* | A change to the RevenueCatUI library |
-  | *refactor* | A code change that neither fixes a bug nor adds a feature |
-  | *style* | Changes that don't affect the meaning of the code (white-space, formatting, missing semi-colons, etc |
-  | *test* | Adding missing tests or correcting existing tests |
-  | *next_release* | Preparing a new release |
-  | *dependencies* | Updating a dependency |
-  | *phc_dependencies* | Updating purchases-hybrid-common dependency |
+  | *breaking* or *pr:breaking* | Changes that are breaking |
```

Comment:
I wonder if we should remove the previous labels from the message, to try to get people to use the new ones...  Though it's true we need to make sure we support the new labels in the automations before adding this? Otherwise, I guess everything would be attributed to the "other changes" section right?

Reply:
Correct, I kept them both because the automations need to be updated first, and since this Dangerfile is shared across all sdks, in case someone is adding a label in a repo that doesn't have the updated label names and automations.

The automations also need to support both for a bit, since they are also used across all repos.

Whenever the automations and labels are in place, we can remove these.